migrations/20240910065643-create-company.js

```diff
@@ -1,34 +0,0 @@
-'use strict';
-/** @type {import('sequelize-cli').Migration} */
-module.exports = {
-  async up(queryInterface, Sequelize) {
-    await queryInterface.createTable('companies', {
-      id: {
-        allowNull: false,
```

Comment:
Why are we allowing null values for company id, any specific reason?

Reply:
ma'am allowNull is false so every user will be having a id .